diff --git a/lib/client.js b/lib/client.js
index c4faeda2e93bd4e6e5c3c2b35becf3504b034f60..8b063693a813c5a9bcdf49eae789ebbf6417e0b2 100644
--- a/lib/client.js
+++ b/lib/client.js
@@ -17605,7 +17605,6 @@ Please report this to https://github.com/markedjs/marked.`, e) {
   function apply2(ctx) {
     try {
       ensureStyle();
-      document.addEventListener("click", onClickCapture, true);
       try {
         window.__DSH_CWD_SESSIONS__ = ctx && ctx.get ? ctx.get("sessions") : void 0;
       } catch {
